ui/tests/integration/helpers/date-format-test.js

```diff
@@ -46,19 +54,26 @@ module('Integration | Helper | date-format', function (hooks) {
   });
 
   test('it supports already formatted dates', async function (assert) {
-    const formattedDate = new Date();
+    const formattedDate = timestamp.now();
     this.set('formattedDate', formattedDate);
 
     await render(hbs`{{date-format this.formattedDate 'MMMM dd, yyyy hh:mm:ss a' isFormatted=true}}`);
-    assert.dom(this.element).includesText(format(formattedDate, 'MMMM dd, yyyy hh:mm:ss a'));
+    assert.dom(this.element).hasText('April 03, 2018 02:15:30 PM');
   });
 
   test('displays time zone if withTimeZone=true', async function (assert) {
     const timestampDate = '2022-12-06T11:29:15-08:00';
-    const zone = new Date().toLocaleTimeString(undefined, { timeZoneName: 'short' }).split(' ')[2];
+    this.set('withTimezone', false);
     this.set('timestampDate', timestampDate);
 
-    await render(hbs`{{date-format this.timestampDate 'MMM d yyyy, h:mm:ss aaa' withTimeZone=true}}`);
-    assert.dom(this.element).hasTextContaining(`${zone}`);
+    await render(
+      hbs`<span data-test-formatted>{{date-format this.timestampDate 'MMM d yyyy, h:mm:ss aaa' withTimeZone=this.withTimezone}}</span>`
```

Comment:
Might have just been due to running this test locally, but I found that it was expecting "CST" and giving "CDT" (in tests run on main as well). Since the time just switched, I figured testing it this way (making sure there's the expected number of characters) ensures that daylight savings doesn't ruin this test in the future. Other suggestions welcome

Reply:
I think this makes sense!